modDbImpl/src/org/aion/db/impl/DatabaseFactory.java

```diff
@@ -166,6 +170,25 @@ private static AbstractDB connectBasic(Properties info) {
                         write_buffer_size,
                         cache_size);
             }
+            case ROCKSDB: {
+                // grab rocksdb specific parameters
+
+                int max_fd_alloc_size = Integer.parseInt(info.getProperty(PROP_MAX_FD_ALLOC, String.valueOf(RocksDBConstants.MAX_OPEN_FILES)));
+                int block_size = Integer.parseInt(info.getProperty(PROP_BLOCK_SIZE, String.valueOf(RocksDBConstants.BLOCK_SIZE)));
+                int write_buffer_size = Integer.parseInt(info.getProperty(PROP_WRITE_BUFFER_SIZE, String.valueOf(RocksDBConstants.WRITE_BUFFER_SIZE)));
+                int read_buffer_size = Integer.parseInt(info.getProperty(PROP_READ_BUFFER_SIZE, String.valueOf(RocksDBConstants.READ_BUFFER_SIZE)));
+                int cache_size = Integer.parseInt(info.getProperty(PROP_CACHE_SIZE, String.valueOf(RocksDBConstants.CACHE_SIZE)));
+
+                return new RocksDBWrapper(dbName,
+                        dbPath,
+                        true,
+                        true,
```

Comment:
why are the database cache and compression always set to true instead of using the config parameters?